### PR TITLE
Remove default user colour schemes

### LINKED ISSUE #113: Remove default user colour schemes
When using the light colour scheme in WP Admin, the Altis logo is barely visible

Screenshot 2019-10-11 at 16 30 18

-----------------

Acceptance criteria:

- [x] unregister default WP admin colour schemes (unless option is already set if possible)
- [x] must allow new custom schemes to be registered
- [x] document how to add a custom colour scheme for accessibility purposes

Optional / extra credit :)

- [ ] add a built in accessible colour scheme
- [ ] ability to enable colour scheme via config, off by default
- [ ] document enabling the colour scheme

**Note**: moved the environment indicator feature to the cloud module https://github.com/humanmade/altis-cloud/issues/124

### PR DESCRIPTION
**Pull request by dashaluna**
Fixes #113

### CODE REVIEW

File: docs/branding.md [hunk 1]
```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
...
+```
+
+- `key`: The unique key for this theme.
+- `name`: The name of the theme.
+- `url`: The URL of the CSS file containing the color scheme.
+- `colors`: An array of CSS color definition strings which are used to give the user a feel for the theme.
+- `icons`: CSS color definitions used to color any SVG icons.
+  - `base`: SVG icon base color.
+  - `focus`: SVG icon color on focus.
+  - `current`: SVG icon color of current admin menu link.
```
Comment (roborourke) [inline docs/branding.md]: Can you add an example of this too, including the `admin_init` action hook? More documentation is better and this doesn't make it clear that you need to use hexadecimal or CSS friendly colour names.

File: docs/branding.md [hunk 2]
```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
 	}
 }
 ```
+
+## Custom admin colour scheme
```
Comment (roborourke) [inline docs/branding.md]: We're using title casing throughout so need to capitalise these words

File: docs/branding.md [hunk 3]
```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
...
+
+To register a custom admin colour scheme add the following function on `admin_init` action.
```
Comment (roborourke) [inline docs/branding.md]: Let's expand on this to say why you'd want to do it - in our case we're recommending it for accessibility purposes.
```suggestion
If the default colour scheme does not meet the accessibility needs of your staff you can register custom schemes to provide high contrast alternatives. To register a custom admin colour scheme add the following function on `admin_init` action:
```

File: inc/branding/namespace.php [hunk 4]
```diff
@@ -76,6 +77,38 @@ function add_color_scheme() {
...
+ * If a user has previously chosen one of the WP default admin color schemes,
+ * leave that scheme as is. Remove all other WP default admin color schemes,
+ * so that they can't be selected.
+ */
+function remove_wp_admin_color_schemes() {
+	global $_wp_admin_css_colors;
+
+	// Use color slugs as keys for a quick access.
+	$admin_color_schemes = [
+		'fresh' => true,
```
Comment (roborourke) [inline inc/branding/namespace.php]: It looks like the value `true` is completely unused here so you don't need an associative array.

File: inc/branding/namespace.php [hunk 5]
```diff
@@ -76,6 +77,38 @@ function add_color_scheme() {
...
+		'sunrise' => true,
+		'ectoplasm' => true,
+		'ocean' => true,
+		'coffee' => true,
+	];
+
+	$user_admin_color = get_user_option( 'admin_color', get_current_user_id() );
+
+	// Remove all WP default admin color schemes, except the one that's selected for a user if any.
+	foreach ( $admin_color_schemes as $color_slug => $color_scheme ) {
```
Comment (roborourke) [inline inc/branding/namespace.php]: `$color_scheme` is never used so as mentioned above, no need for an associative array.

File: docs/branding.md (hunk 2 above)
Comment (dashaluna) [reply docs/branding.md]: But `## Login screen` above is in low-case. I copied that. Must be missing something.
Do you mean `Custom Admin Colour Scheme`?

Comment (roborourke) [reply docs/branding.md]: Right ok, that's a mistake then sorry! We need to go through and clarify all that and make it consistent as part of a general documentation sweep

File: docs/branding.md [hunk 6]
```diff
@@ -19,3 +19,37 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
 	}
 }
 ```
+
+## Custom Admin Colour Scheme
```
Comment (roborourke) [inline docs/branding.md]:
```suggestion
## Custom Admin Colour Scheme

```

File: docs/branding.md [hunk 7]
```diff
@@ -19,3 +19,37 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
...
+If the default colour scheme does not meet the accessibility needs of your staff you can register custom schemes to provide high contrast alternatives. To register a custom admin colour scheme add the following function on `admin_init` action.
```
Comment (roborourke) [inline docs/branding.md]:
```suggestion
If the default colour scheme does not meet the accessibility needs of your staff you can register custom schemes to provide high contrast alternatives. To register a custom admin colour scheme add the following function on the `admin_init` action.
```

**Pull request merged**